io.edgehog.devicemanager.AgentInfo.json

```diff
@@ -0,0 +1,24 @@
+{
+  "interface_name": "io.edgehog.devicemanager.AgentInfo",
```

Comment:
@rbino about naming: should we keep `AgentInfo` or change it to a more generic `RuntimeInfo`?

Reply:
I'd change it to `RuntimeInfo`, `Agent` is already used for a specific pairing actor (the one that registers devices) and using it here might introduce confusion in the naming